shell/platform/android/io/flutter/Log.java

```diff
@@ -0,0 +1,86 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
How much binary size does this add to the engine?

Have we considered implementing tracing at the Android level rather than making it easier to add more logs?

Reply:
I did consider direct logging, which would litter the code with if statements about build config. I think this is a significantly more readable alternative.

As for binary size, I'm pretty sure the binary size impact of this one class is less than if we surrounded every log statement with an if-statement. But I'm also fairly confident that both approaches are negligible.

Reply:
The if debug gets compiled out since it's a statically known by the compiler.  This won't

Reply:
This added ~700 bytes compressed to the release build of flutter.jar

Reply:
I'll look for another way. I really, really don't like the idea of dropping if-statements everywhere.